### PR TITLE
clean up language a bit

### PR DESCRIPTION
**Pull request by phrawzty**
`r?`

### CODE REVIEW

File: docs/tutorial.rst [hunk 1]
```diff
@@ -122,10 +123,10 @@ Let us fetch our new collection of tasks:
 
 
 Keep a note of the ``ETag`` and of the ``last_modified`` values
-returned (here both ``"1436171996916"``), we'll need them for a later
+returned (here both "``1436171996916``") - we'll need them for a later
```
Comment (leplatrem) [inline docs/tutorial.rst]: Careful, double quotes are parts of the value

**Comment by ametaireau** [comment]
This is so helpful to have someone speaking correct English! <3

**Comment by leplatrem** [comment]
> speaking correct English

correctly speaking English ? #kiddin

**Comment by Natim** [comment]
r+ :+1:

**Pull request merged**